README.wcf.md

```diff
@@ -42,14 +42,12 @@ docker run --name wcfclientsample --rm -it -e HOST=172.26.236.119 mcr.microsoft.
 
 ### Version Compatibility
 
-Version Tag | OS Version | Supported .NET Versions
--- | -- | --
-4.8.1 | windowsservercore-ltsc2022 | 4.8.1
-4.8 | windowsservercore-ltsc2022, windowsservercore-ltsc2019, windowsservercore-ltsc2016 | 4.8
-4.7.2 | windowsservercore-ltsc2019, windowsservercore-ltsc2016 | 4.7.2
-4.7.1 | windowsservercore-ltsc2016 | 4.7.1
-4.7 | windowsservercore-ltsc2016 | 4.7
-4.6.2 | windowsservercore-ltsc2016 | 4.6.2
+If you created your app using an earlier version of .NET Framework, you can generally upgrade it to .NET Framework 4.8+ easily.
+Additionally, .NET Framework 4.8 and 4.8.1 can run apps that were built targeting any version of .NET Framework 4.
```

Comment:
This isn't guaranteed. There may be incompatibilities. I think we at least need a qualifier here:

```suggestion
Additionally, .NET Framework 4.8 and 4.8.1 can typically run apps that were built targeting any version of .NET Framework 4.
```